data-products/src/main/scala/org/sunbird/analytics/job/report/StateAdminReportJob.scala

```diff
@@ -141,7 +141,7 @@ object StateAdminReportJob extends optional.Application with IJob with StateAdmi
                 col("decrypted-phone").as("Phone number"),
                 col("decrypted-email").as("Email ID"),
                 col("Diksha Sub-Org ID"),
-                col("provider").as("Channel"))
+                col("originalprovider").as("Channel"))
```

Comment:
We have a org table joined as well. Why can't we leverage channel from there, instead of provider or original_provider?

I am not sure if it is good to have footing on provider and original_provider, esp when we want to unify these fields.

Also, am wanting to check why the provider turns upper case when passed from UI. Perhaps a fix lies there too.

Reply:
I hope we are on same page on getting the value from field originalprovider.

Reply:
Okay for now, we need to get away from these original_columns - that's probably a simple direction.